admin/javascript/src/state/records/actions.js

```diff
@@ -32,7 +32,11 @@ export function fetchRecords(recordType, method, url) {
       type: ACTION_TYPES.FETCH_RECORDS_REQUEST,
       payload,
     });
-    return backend[method.toLowerCase()](populate(url, payload), {}, headers)
+    const args = method.toLowerCase() === 'get'
```

Comment:
In a REST sense, yes it would always be GET. But from a generic API handling perspective, I wouldn't make that assumption.